psi4/src/psi4/libfock/jk.h

```diff
@@ -1064,6 +1087,10 @@ class PSI_API MemDFJK : public JK {
     */
     void print_header() const override;
 
+    void set_omega_alpha(double alpha);
+    void set_omega_beta(double beta);
+    void set_wcombine(bool wcombine);
```

Comment:
these aren't inherited from JK?

Reply:
2 reasons why these have to be done differently:

1. The actual math is done by the dfhelper object in MemDFJK, so inheriting a function won't change the values there, and I can't program altering values in a dfhelper object in a generic JK object because it doesn't have one.

2. wcombine is only available on MemDFJK right now, so I need to the JK object to throw an error if the user tries to use wcombine on another JK daughter class.

Reply:
Maybe I'm really missing the connection. There's two definitions -- one in header for JK class and one in dfhelper header -- but the one for MemDFJK class has only the declaration. where am I missing its implementation?

Reply:
Oversight on my part

Reply:
Next commit should have the definitions added in `MemDFJK.cc`

Reply:
My worry then would be why was this working w/o the definitions. You've run the combinations of wcombine=T/F, and scf_type=mem_df/disk_df to make sure they're taking the code paths you expect?

Reply:
The values were getting set to the dfhelper object in `jk.preiterations()` which is called python side in `jk.initialize()`